client/js/components/scroll-to-info.js

```diff
@@ -0,0 +1,27 @@
+import anim from 'anim';
+import { getDocumentHeight, getWindowHeight } from '../util';
+
+export default (el) => {
+  const idToScrollTo = el.getAttribute('data-element');
```

Comment:
Could we not get this from the `href`?
We could just have (paracoded):

HTML
```HTML
<a href="#scrolltoid" class="js-scroll-to">Bloop</a>
```

JS
```JS
const el = document.querySelector('.js-scroll-to');
scrollTo(document.getElementById(el.href));
```
?